src/components/structures/RoomView.tsx

```diff
@@ -1752,7 +1752,6 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
     private get messagePanelClassNames(): string {
         return classNames("mx_RoomView_messagePanel", {
             mx_IRCLayout: this.state.layout === Layout.IRC,
-            mx_GroupLayout: this.state.layout === Layout.Group,
```

Comment:
This was Introduced for search result in bubble message layout [here](https://github.com/matrix-org/matrix-react-sdk/pull/7648/files#diff-13590ebbbcd62256ff3ead94de2e6be41867b82ad9e0470c4b63ff63c1f5f78bR1789) to overcome `:not([data-layout=bubble])`.